client/components/events/EventListView/EventListView.tsx

```diff
@@ -0,0 +1,53 @@
+import  { EventListCard } from './EventListCard'
+import { Id, Event } from '../../../context'
+import styled from 'styled-components'
+import { colours, device, fontWeight, desktopFontSize, mobileFontSize } from '../../../styles'
+
+interface EventViewProps {
+  filteredEvents: Array<Event>
+}
+
+const ListView = styled.div`
+  width: 100%;
+`
+
+const ListViewRow = styled.div`
+  display: flex;
+  @media ${device.tablet} {
+    display: block;
+  }
+`
+
+const ListViewDate = styled.div`
+  font-weight: bold;
+  font-size: ${desktopFontSize.subtitle2};
+  flex: 0 0 200px;
+  color: ${colours.neutralDark1};
+  @media ${device.tablet} {
+    margin-bottom: 15px;
+  }
+`
+
+export const EventListView = ({ filteredEvents }: EventViewProps) => {
+  
+  const getDateLabelIfNecessary = (index: number, allEvents: Event[]) => {
+    if(index == 0 || !(allEvents[index-1].startDate.isSame(allEvents[index].startDate, 'day'))){
+      //Format: THU, NOV 17TH
+      return allEvents[index].startDate.format('ddd, MMM Do').toLocaleUpperCase();
+    }
+    return null;
+  }
+
+  return (
+    <ListView> 
+      {filteredEvents.map((event, index, allEvents) => (
+        <ListViewRow key={`list-view-row-${event.id}`}>
+          <ListViewDate>
+            {getDateLabelIfNecessary(index, allEvents)}
+          </ListViewDate>
+          <EventListCard key={`event-list-card-${event.id}`} event={event} />
+        </ListViewRow>
+      ))}
+    </ListView>
```

Comment:
That's one way to do it. Here's another:

```jsx
<ListView>
  // for each day:
  <ListViewSection>
    <ListViewDate>{day.formatHoweverYouWant()}</ListViewDate>
    <ListViewItems>
      {allEventsOn(day).map((event) => <EventListCard ... event={event}/>)}
    </ListViewItems>
  </ListViewSection>
</ListView>
```

Then you can use media queries to change the flex-direction of `ListViewSection`, which determines whether the date shows up on top or to the side. The advantage is that we only create a `ListViewDate` when we actually intend to render some text.